pithos/plugins/notify.py

```diff
@@ -77,3 +79,47 @@ def on_disable(self):
         if self._shutdown_handler:
             self._app.disconnect(self._shutdown_handler)
             self._shutdown_handler = 0
+
+
+class NotifyPluginPrefsDialog(Gtk.Dialog):
+    __gtype_name__ = 'NotifyPluginPrefsDialog'
+
+    def __init__(self, window, settings):
+        super().__init__(use_header_bar=1)
+        self.set_title(_('Notification Preferences'))
+        self.set_default_size(300, -1)
+        self.set_resizable(False)
+        self.connect('delete-event', self.on_close)
+
+        self.pithos = window
+        self.settings = settings
+        self.show_album = self.settings['data'] == 'True' if self.settings['data'] else False
+
+        box = Gtk.Box()
```

Comment:
It looks very squished together, you should set the margin. Like `6` all around maybe?

Reply:
The spacing helped but I just added a pair of new-lines to the label. I think it helps how it looks?